src/ext/java/org/opentripplanner/ext/fares/model/FareAttribute.java

```diff
@@ -38,12 +39,25 @@
     this.youthPrice = builder.youthPrice();
     this.seniorPrice = builder.seniorPrice();
     this.journeyDuration = builder.journeyDuration();
+    this.agency = builder.agency();
   }
 
   public static FareAttributeBuilder of(FeedScopedId id) {
     return new FareAttributeBuilder(id);
   }
 
+  public boolean isAgencySet() {
+    return agency != null;
+  }
+
+  public String getAgency() {
+    return agency;
+  }
+
+  public void setAgency(String agency) {
```

Comment:
Is the value the id of the agency? If yes you should pass a feed-scoped id.

Reply:
Sorry for the long delay - would this only be limited to this particular parameter or should the agency field of the FareAttribute class be a FeedScopeId as well?

Reply:
I'd would be totally nice if you would change all the existing fields as well, but since you're a relative newcomer we will not insist on it.

Reply:
Ok, I think I will then take the easy option and only change the parameter type